en/docs/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/managing-apis-in-kubernetes.md

```diff
@@ -0,0 +1,168 @@
+# Managing APIs in Kubernetes
+
+WSO2 API Controller, **apictl** allows you to manage APIs in Kubernetes with the use of WSO2 API operator. This approach uses the kubernetes native way to deploy the APIS into a running microgateway cluster from the command line.
+
+!!! Prerequisites
+
+    - [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+    - [Kubernetes v1.14 or above](https://Kubernetes.io/docs/setup/) <br>
+
+        - Minimum CPU : 2vCPU
+        - Minimum Memory : 2GB
+
+!!! info
+    **Before you begin** 
+
+    -  First download the microgateway distribution from the [github release page](https://github.com/wso2/product-microgateway/releases/) and extract that to a folder of your choice. Then deploy microgateway in kubernetes cluster. Please refer [Steps for deploying  microgateway in kubernetes]()
```

Comment:
Don't we need to add the docs link after this for the steps for the mentioned deployment?
**[Steps for deploying microgateway in kubernetes]**

Reply:
And Shall we change the reference subject as  "Steps for deploying  Microgateway in Kubernetes" as @wasuradananjith mentioned.

Reply:
Fixed with 21f466ce5d99225a24a45351eae0997e87f8c2f5